rpms/base.pan

```diff
@@ -2,16 +2,49 @@ unique template rpms/base;
 
 variable OS_CORE_ONLY ?= false;
 variable OS_CORE_ISCSI_ENABLED ?= false;
+variable OS_CORE_RDMA_ENABLED ?= false;
+
 
 # Standard YUM groups for all machines
-include {'rpms/group/core' };
-include { if ( ! OS_CORE_ONLY ) 'rpms/group/base' };
-include { if ( ! OS_CORE_ONLY ) 'rpms/management-utils' };
+
+'/software/groups' = {
+  # Default packages are mainly firmware for Wifi
+  SELF['core'] = dict('mandatory', true,
+                      'default',false,
+                      'optional', false,
+                     );
```

Comment:
Again, I don't have strong opinion about what is the best formatting. But the one used is consistent with what is used everywhere in the template library... I don't mind to change but I would prefer this is discussed at a workshop or on the mailing list before...